go/src/koding/config/genconfig.go

```diff
@@ -0,0 +1,233 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+var (
+	input  = flag.String("i", "-", "")
+	output = flag.String("o", "-", "")
+)
+
+var t = template.Must(template.New("").Parse(`// This file is auto-generated. DO NOT EDIT!
```

Comment:
i am not sure why we need this template and all the file and template generation. could you please explain? You are already generating a json file  in `goGenerate`, why not just use json encode/decode? is not it simpler? we are already generating the config for each env while deploying